planner/optimize.go

```diff
@@ -200,6 +211,27 @@ func handleInvalidBindRecord(sctx sessionctx.Context, level string, bindRecord b
 	globalHandle.AddDropInvalidBindTask(&bindRecord)
 }
 
+func handleEvolveTasks(ctx context.Context, sctx sessionctx.Context, br *bindinfo.BindRecord, stmtNode ast.StmtNode, planHint string) {
```

Comment:
Need we handle prepare&&execute stmt here?

Reply:
Seems like we don't support SQL bind for `prepare / execute` according to `extractSelectAndNormalizeDigest`.

Reply:
Nice catch, it actually cannot handle prepare stmt currently, because we cannot restore the arguments now.